mrtt-ui/src/components/CheckboxGroupMangroveWithLabel.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
```

Comment:
This reusable checkbox group component assumes react hook form, and uses the a input label to match those of the rest of the form inputs

Reply:
This component is where we glue in react hook form